src/components/v2/V2Project/LaunchProjectPayerModal.tsx

```diff
@@ -32,27 +42,169 @@ export default function LaunchProjectPayerModal({
 }: {
   visible: boolean
   onClose: VoidFunction
-  useDeployProjectPayerTx: () => TransactorInstance<{}> | undefined
+  useDeployProjectPayerTx: () =>
+    | TransactorInstance<DeployProjectPayerTxArgs>
+    | undefined
   onConfirmed?: VoidFunction
 }) {
+  const { userAddress } = useContext(NetworkContext)
+  const { tokenAddress } = useContext(V2ProjectContext)
+
   const [loadingProjectPayer, setLoadingProjectPayer] = useState<boolean>()
   const [transactionPending, setTransactionPending] = useState<boolean>()
   const [projectPayerAddress, setProjectPayerAddress] = useState<string>()
+  const [activeKey, setActiveKey] = useState<number>()
+
+  const [customBeneficiaryEnabled, setCustomBeneficiaryEnabled] =
+    useState<boolean>(false)
+  const [tokenMintingEnabled, setTokenMintingEnabled] = useState<boolean>(true)
+  const [preferClaimed, setPreferClaimed] = useState<boolean>(false)
+
+  const [form] = useForm<{
+    memo: string
+    customBeneficiaryAddress: string
+  }>()
 
   const [confirmedModalVisible, setConfirmedModalVisible] = useState<boolean>()
   // TODO: load project payer and show different thing in this section if the project already has one
   // (Issue: #897)
 
   const deployProjectPayerTx = useDeployProjectPayerTx()
 
+  const advancedSettingsMargin = '20px'
+  const switchMargin = '20px'
+
+  function AdvancedOptionsCollapse() {
+    return (
+      <Collapse style={{ border: 'none' }} activeKey={activeKey}>
+        <CollapsePanel
+          header={
+            <span onClick={() => setActiveKey(activeKey === 0 ? undefined : 0)}>
+              <Trans>Advanced options (optional)</Trans>
+            </span>
+          }
+          key={0}
+          style={{ border: 'none', marginLeft: '-18px' }}
```

Comment:
**question** is there a way we can style this without negative margins? Negative margins should be avoided.

Reply:
In this case, it's a lot more convenient. I believe the only way we could style what we need to get the current layout would be to add classes and custom CSS - because antd doesn't allow styling of these particular elements

Reply:
Fair call, tahnks for checking - lets ship it